lib/dl_file_uploader_lib/dl_file_uploader_lib/yadocuments_client.py

```diff
@@ -0,0 +1,43 @@
+import requests
+
+
+class YaDocumentsClient:
+    headers = {
+        "Content-Type": "application/json",
+        "Accept": "application/json",
+    }
+
+    def get_spreadsheet_public(self, link):
+        resp = requests.get(
+            f"https://cloud-api.yandex.net/v1/disk/public/resources/download/?public_key={link}",
```

Comment:
Let's make the common part into a class variable

Reply:
what common part do you mean?

Reply:
`https://cloud-api.yandex.net/v1/disk/`